GreenGarbageApp/app/src/main/java/com/example/greengarbageapp/activities/MainActivity.kt

```diff
@@ -41,6 +46,32 @@ class MainActivity : AppCompatActivity(){
                 drawerLayout.setDrawerLockMode(DrawerLayout.LOCK_MODE_LOCKED_CLOSED)
             }
         }
+        // QR-code scanner
```

Comment:
Why is this in MainActivity?

Reply:
I have looked into it and I realized it is skipping our intro page and going directly to game page. I am moving this to other class as you suggested.